src/main/resources/application.yaml

```diff
@@ -14,3 +14,6 @@ management:
 spring:
   application:
     name: Reform Scan Notification Service
```

Comment:
missing data source config

Reply:
Will add it

Reply:
still missing some hystrix stuff there

Reply:
Netflix Hystrix is now officially in maintenance mode so better not to include it for new projects ?

Reply:
woops. sorry, hikari, not hystryx. don't know what why i said

Reply:
yes best to include that as we know default doesn't work. Can do a separate PR 🤷‍♂

Reply:
Added hikari config